### ISSUE TITLE
Evaluate Model Performance

### ISSUE THREAD

**Issue by saadahmad22** [saadahmad22/dynamic-gesture-recognition#4]
- Design the loss function
- Fine-tune as needed
